decidim-dev/lib/decidim/dev.rb

```diff
@@ -14,5 +14,25 @@ module Dev
     def self.asset(name)
       File.expand_path(File.join(__dir__, "dev", "assets", name))
     end
+
+    # Public: add rake tasks
+    def self.install_tasks
+      Dir[File.join(__dir__, "../tasks/*.rake")].each do |file|
+        load file
+      end
+    end
+
+    # Public: Sets the dummy application path for testing.
+    #
+    # path - A string value defining the path.
+    def self.dummy_app_path=(path)
+      @dummy_app_path = path
+    end
+
+    # Public: Get the dummy application path and raises an error if it is not set.
+    def self.dummy_app_path
+      raise StandardError, "You need to define the `dummy_app_path` first." unless @dummy_app_path
```

Comment:
Add a more helpful message. SOmething like: 

"Please, add `Decidim::Dev::dummy_app_path = File.expand_path(File.join("..", "spec", "decidim_dummy_app"))` to your spec helper with the path to generate the dummy app"